web_src/js/features/common-organization.js

```diff
@@ -1,14 +1,15 @@
-import $ from 'jquery';
 import {initCompLabelEdit} from './comp/LabelEdit.js';
 import {toggleElem} from '../utils/dom.js';
 
 export function initCommonOrganization() {
-  if ($('.organization').length === 0) {
+  if (!document.querySelectorAll('.organization')) {
```

Comment:
Thanks, fixed.

Reply:
Actually, as a refactoring, it could keep the old logic, or it could also remove this `if` check. Because there is a new `if` check below.

Either is fine to me.

Reply:
Yes, it makes sense to remove it.